lib/tasks/import.rake

```diff
@@ -86,7 +86,7 @@ namespace :import do
       counter += 1
       print "Completed #{counter}\r"
 
-      MetadataTools.add_dataset_metadata(obj, orgs_cache, theme_cache)
+      MetadataTools.save_to_postgres(obj, orgs_cache, theme_cache)
```

Comment:
how about just `persist`?

Reply:
@laurentqro - Yep - I can change 'save_to_postgres' to 'persist' and might as well change 'save_to_elasticsearch' to 'index'